### PR TITLE
added Requests model

### PR DESCRIPTION
**Pull request by c-s-ale**
### What this PR does (required):
Adds the model for requests as outlined by the linked issue

### Screenshots / Videos (required):
N/A 

### Any information needed to test this feature (required):
N/A

### Any issues with the current functionality (optional):
Only the models are set up, once the model gets approved I will generate the routes!

### CODE REVIEW

File: server/models/Requests.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  sitter_id: {
+    type: String,
+    unique: true,
+  },
```
Comment (moffatethan) [inline server/models/Requests.js:12]: I would utilize mongoose `ObjectId` type here as opposed to string, also conventionally JavaScript does not do snake case but instead make use of camelCase.

Example: 
```js
user: {
  type: mongoose.Types.ObjectId,
  required: true,
  ref: 'user'
}
```

Comment (moffatethan) [reply server/models/Requests.js:12]: You can follow the same for sitter as well.

**Review by moffatethan — APPROVED** [review]
I'm going to approve this and if you could make the changes I commented on then merge it in that'd be great!

File: server/models/Requests.js (hunk 1 above)
Comment (c-s-ale) [reply server/models/Requests.js:12]: This is an excellent recommendation, I have made those changes!

**Pull request merged**